x-pack/solutions/security/plugins/security_solution/public/common/components/user_privileges/user_privileges_context.tsx

```diff
@@ -48,11 +51,17 @@ export const UserPrivilegesProvider = ({
   const crud: boolean = kibanaCapabilities[SECURITY_FEATURE_ID].crud === true;
   const read: boolean = kibanaCapabilities[SECURITY_FEATURE_ID].show === true;
```

Comment:
Shall we rename `crud` and `read` to `crudSiem` and `readSiem` for consistency with the rules capabilities?

Reply:
We could, but that would further increase the scope of this PR as _anything_ relying on those privileges would need to change. I agree that it is inconsistent, but I don't think it's prudent to address right now.